### PR TITLE
adding seeds

### PR DESCRIPTION
**Pull request by dumboluzz**
Adding seed:
- 4 users (email: firstname@lastname.com, password: 123456)
- 10 alpacas
- 20 bookings

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,41 @@
...
 #   Character.create(name: 'Luke', movie: movies.first)
+
+puts "-- clearing database --"
+Booking.destroy_all
+Alpaca.destroy_all
+User.destroy_all
+
+puts "-- adding default users --"
+User.create(first_name: "Vincent", last_name: "Stuber", email: "vincent@stuber.com", password: "123456", password_confirmation: "123456")
+User.create(first_name: "Sopie", last_name: "Sorin", email: "sophie@sorin.com", password: "123456", password_confirmation: "123456")
```
Comment (sophiecs333) [inline db/seeds.rb:16]: Hey! It's Sophie not Sopie!

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,41 @@
...
+  a.wool_type = ["Suri", "Huacaya"].sample
+  a.owner = User.order(Arel.sql('RANDOM()')).first
+  a.save
+end
+
+puts "-- generating bookings --"
+20.times do
+  b = Booking.new
+  b.start_date = Date.today + 1
+  b.end_date = Date.today + 5
```
Comment (sophiecs333) [inline db/seeds.rb:39]: same here for the ending date

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,41 @@
...
+puts "-- clearing database --"
+Booking.destroy_all
+Alpaca.destroy_all
+User.destroy_all
+
+puts "-- adding default users --"
+User.create(first_name: "Vincent", last_name: "Stuber", email: "vincent@stuber.com", password: "123456", password_confirmation: "123456")
+User.create(first_name: "Sopie", last_name: "Sorin", email: "sophie@sorin.com", password: "123456", password_confirmation: "123456")
+User.create(first_name: "Maria", last_name: "Pigazzi", email: "maria@pigazzi.com", password: "123456", password_confirmation: "123456")
+User.create(first_name: "Finn", last_name: "Stürenberg", email: "finn@stuerenberg.com", password: "123456", password_confirmation: "123456")
```
Comment (finnstue) [inline db/seeds.rb:18]: Hey! It's Stürenburg not Stürenberg!

**Comment by finnstue** [comment]
Please add these for the Nickname:
Faker::GreekPhilosophers.name
Faker::Science.scientist
Faker::Superhero.name
Faker::Fantasy::Tolkien.character
Faker::Creature::Dog.name
Faker::Creature::Cat.name
Faker::Movies::HarryPotter.character
Faker::Games::Zelda.character

**Review by finnstue — APPROVED** [review]

**Pull request merged**